### ISSUE TITLE
The mean pixel in config.py file

### ISSUE THREAD

**Issue by zimenglan-sysu-512** [rbgirshick/py-faster-rcnn#37]
I noticed line 47 of config.py the code is:
     # These are the values originally used for training VGG16
     __C.PIXEL_MEANS = np.array([[[102.9801, 115.9465, 122.7717]]])
But the original VGG16 trained model on model zoo is [103.939, 116.779, 123.68]
Source: https://gist.github.com/ksimonyan/211839e770f7b538e2d8#file-readme-md  (linked from author’s project page).
I know the difference is small, and probably doesn’t have a significant impact. But the fact that these numbers are not the same here may indicate the original VGG16 on model zoo is different from the one in Faster RCNN. But Faster RCNN claim it was the original for VGG16. I know in the VGG16 paper there are different kinds of 16 layer models and maybe it uses a different one? But those other 16 layer models aren’t published in their original form the best of my knowledge. Anyway, this inconsistency may mean other things are inconsistent.
Thanks.

**Comment by rbgirshick** [comment]
Hmm, this is odd. I definitely haven't trained my own VGG16 (aka VGG-D); I used the one that Karen posted online, downloaded some time ago. I'm not sure now where the 102.9801, ... BGR mean pixel values came from. I either accidentally copied them from somewhere else or, maybe, I computed mean values from PASCAL images (though I don't recall doing that). Either way, I doubt that this matters in practice. In fact, I'm using the same mean values for VGG16, CaffeNet, and VGG_CNN_M_1024 even though the latter two were pre-trained with **mean images**. If you're concerned about the model being different from the one that Karen released, I suggest checking the weights.
